.github/workflows/ci_tests.yaml

```diff
@@ -120,6 +120,46 @@ jobs:
         cat operator.log
         echo "::endgroup::"
 
+  e2e-openshift-tests:
+          name: Run e2e Openshift tests
+          runs-on: ubuntu-latest
+      
+          steps:
+          - uses: actions/checkout@v3
+      
+          - uses: actions/setup-go@v3
+            with:
+              go-version-file: go.mod
+      
+          - name: Download binaries
+            run: |
+              make download-binaries
+      
+          - name: Create K8s KinD Cluster
+            run: |
+              make test-cluster
+              make test-openshift-setup
+      
+          - name: Build and Push Test Container Image to KIND node
+            run: |
+              make docker-build IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+              kind load docker-image localhost:5000/gatekeeper-operator:$GITHUB_SHA
+      
+          - name: E2E Openshift Tests
+            run: |
+              make deploy-ci NAMESPACE=openshift-gatekeeper-system IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+              kubectl -n openshift-gatekeeper-system patch deployment gatekeeper-operator-controller -p '{"spec" : {"template" :{"spec" : {"securityContext" :{"seccompProfile" : {"type": "RuntimeDefault"}}}}}}' 
```

Comment:
What happens if you don't patch this?

Reply:
Because I installed  https://raw.githubusercontent.com/openshift/api/release-4.15/route/v1/route.crd.yaml